cmd/top-drives-spinner.go

```diff
@@ -140,20 +140,22 @@ type driveIOStat struct {
 }
 
 func generateDriveStat(disk madmin.Disk, curr, prev madmin.DiskIOStats, interval uint64) (d driveIOStat) {
-	d.endpoint = disk.Endpoint
-	d.used = 100 * disk.UsedSpace / disk.TotalSpace
-	d.util = 100 * float64(curr.TotalTicks-prev.TotalTicks) / float64(interval)
-	currTotalIOs := curr.ReadIOs + curr.WriteIOs + curr.DiscardIOs
-	prevTotalIOs := prev.ReadIOs + prev.WriteIOs + prev.DiscardIOs
-	totalTicksDiff := curr.ReadTicks - prev.ReadTicks + curr.WriteTicks - prev.WriteTicks + curr.DiscardTicks - prev.DiscardTicks
-	if currTotalIOs > prevTotalIOs {
-		d.tps = currTotalIOs - prevTotalIOs
-		d.await = float64(totalTicksDiff) / float64(currTotalIOs-prevTotalIOs)
+	if disk.TotalSpace > 0 {
```

Comment:
We can just keep to the left and return early.
```suggestion
	if disk.TotalSpace < 0 {
		return d	
	}
```